http.md

```diff
@@ -0,0 +1,342 @@
+  - `slot`: `<u64>`  The slot this bundle was processed in.
+  - `confirmationStatus`: `<string|null>` - The bundle transaction's cluster confirmation status; Either processed, confirmed, or finalized. See [Commitment](#configuring-state-commitment) for more on optimistic confirmation.
+  - `meta`: `<object|null>`: Transaction status metadata object:
+    - `err`: `<object|null>`: Error if the bundle failed, null if the bundle succeeded/finalized. The error reasons could be a bundle acceptance failure like not enough tips or failure of one the included transactions, for example blockhash expired. The transaction failure errors can be found at [Transaction errors](https://docs.rs/solana-sdk/latest/solana_sdk/transaction/enum.TransactionError.html#). If a transaction error, the transaction id will also be included to identify which transaction caused the failure.
+
+### Code sample
+
+### Request
+
+```bash
+curl http://localhost:8899 -X POST -H "Content-Type: application/json" -d '
+{
+    "jsonrpc": "2.0",
+    "id": 1,
+    "method": "getBundleStatuses",
+    "params": [
+        [
+            "3Eq21vXNB5s86c62bVuUfTeaMif1N2kUqRPBmGRJhyTA"
+        ]
+    ]
+}
+'
+```
+
+### Response
+
+```json
+{
+    "jsonrpc": "2.0",
+    "result": {
+        "context": {
+            "slot": 82
+        },
+        "value": [
+            {
+                "bundle": {
+                    "signatures": [
+                        "3Eq21vXNB5s86c62bVuUfTeaMif1N2kUqRPBmGRJhyTA",
+                        "2nBhEBYYvfaAe16UMNqRHre4YNSskvuYgx3M6E4JP1oDYvZEJHvoPzyUidNgNX5r9sTyN1J9UxtbCXy2rqYcuyuv"
+                    ]
+                },
+                "slot": 1234,
+                "confirmationStatus": "finalized",
+                "err": "null"
+            }
+        ]
+    },
+    "id": 1
+}
+```
+
+Sample error codes:
+
+```json
+{
+    "results": [
+        {
+            "err": {
+                "code": -32002,
+                "message": "Transaction simulation failed: Blockhash not found",
+                "transaction": {
+                    "signatures": [
+                        "2nBhEBYYvfaAe16UMNqRHre4YNSskvuYgx3M6E4JP1oDYvZEJHvoPzyUidNgNX5r9sTyN1J9UxtbCXy2rqYcuyuv"
+                        ]
+                }
+            }
+        },
+        {
+            "err": {
+                "code": -42000,
+                "message": "Bundle failed: Not enough tips"
+            }
+        },
+    ]
+}
+
+```
+
+## getBundlesForAddress
+Returns signatures for confirmed bundles that include the given address in their accountKeys list. Returns signatures backwards in time from the provided signature or the most recent confirmed block.
+
+### Parameters:
+
+`<string>`: `required` Account address as a base-64 encoded string.
```

Comment:
addresses are typically base58... yeah i know its confusing lol

Reply:
Is bundle_id base-58 as well? transaction id is base-58